src/components/transverse/MenuLateral/MenuLateral.tsx

```diff
@@ -93,7 +93,7 @@ export default function MenuLateral({ children }: Readonly<PropsWithChildren>):
                   <Fragment key={menu.url}>
                     <li className={`fr-sidemenu__item ${activeClass}`}>
                       <Link
-                        aria-controls={menu.ariaControls ?? ''}
+                        aria-controls={menu.ariaControls ?? undefined}
```

Comment:
et juste comme ça ❓ 

```suggestion
                        aria-controls={menu?.ariaControls}
```

Reply:
@Ornella452 
c'est plutôt 
```suggestion
                        aria-controls={menu.ariaControls}
```